src/main/model/URLCommand.java

```diff
@@ -0,0 +1,76 @@
+package main.model;
+//not sure why we have this URL when there is one above it in the Curl class
+public class URLCommand {
+    private String URLs;
+    //not sure why we have this URL when there is one above it in the Curl class
+    private String company;
```

Comment:
You have an entity field in another file, is this actually a company, or is it an entity? Always good to hold consistency.